runtime/src/main/java/org/corfudb/runtime/view/StreamsView.java

```diff
@@ -74,6 +77,7 @@ public IStreamView getUnsafe(UUID stream) {
     public IStreamView get(UUID stream, StreamOptions options) {
         IStreamView streamView = runtime.getLayoutView().getLayout().getLatestSegment()
                 .getReplicationMode().getStreamView(runtime, stream, options);
+        streamCache.put(stream, streamView);
```

Comment:
This line was mistakenly removed in the compactionMark commit, please re-add this line in that commit instead of fixing it in this cache trim commit.

Reply:
done.